### DISCUSSION TITLE
Box86 + Freebox Delta VM ?

### DISCUSSION THREAD

**Discussion by nitronixxx** [ptitSeb/box86#415] [Q&A]
Hi, can i install Box86 on Debian hosted on a Freebox Delta VM ? I want to try to host a Steam dedicated server for
Half-Life 2 Deathmatch.... Is that possible ?

**Comment by ptitSeb** [comment]
I have no idea, but you can probably try to build box86 yes. If it's debian based, box86 should build.
What cpu it's using?

**Reply by nitronixxx** [reply]
The CPU : ARMv8, Armada 8040 (88F8040)  Cortex-A72
